### ISSUE TITLE
pretrained models

### ISSUE THREAD

**Issue by devinxzhang** [MIC-DKFZ/medicaldetectiontoolkit#10]
Are there any pretrained model available? especially 3D models. T.I.A.

**Comment by pfjaeger** [comment]
Not yet. What model would you be interested in?

**Comment by devinxzhang** [comment]
@pfjaeger 3D models. Basically, 3D backbones (e.g. 3D res50, res101). Thus we can train different models like 3D mask rcnn. Thanks.

**Comment by pfjaeger** [comment]
What data set would you like them to be pretrained on ?

**Comment by devinxzhang** [comment]
@pfjaeger LIDC is fine. Thanks.

**Comment by pfjaeger** [comment]
Sorry for the wait, I am now back and willing to train Mask R-CNN on the entire LIDC data set ( so far I have only run cross validations with 60% training data) to upload the parameters. However, have you considered, that unlike in the ImageNet case, here we can not pretrain on a generic task like whole image classification, but on a specific task like lesion detection + categorization. Hence, generalization to other tasks might be compromised. Moreover, I would only recommend to use this if your target domain is CT as well. Could you please confirm your interest?

**Comment by devinxzhang** [comment]
@pfjaeger Thanks for your reply. I have trained the 3D Mask R-CNN model with LUNA dataset which is a subset of LIDC with 888 CT scans from scratch. 60% for train, 20% for val, and 20% for test. However, I only achieved th AP of about 0.63. Most of my implements were the same to yours, except that I resampled CT scans to [1,1,1]. So the anchors size is the same in all three dimensions.

**Comment by pfjaeger** [comment]
is 0.63 a bad score? whats your benchmark?
